includes/models/model.llms.quiz.php

```diff
@@ -30,6 +30,7 @@
  * @since 4.0.0 Remove deprecated methods.
  * @since 4.2.0 Added a parameter to the `is_orphan()` method to deeply check the quiz is not really attached to any lesson.
  * @since 5.0.0 Remove previously deprecated method `LLMS_Quiz::get_lessons()`.
+ * @since [version] Add `get_questions_count()` function.
```

Comment:
```suggestion
```
not needed.
I know the other changelog items are confusing you but we stopped logging changes in the class dockblock except for big changes that are not tracked elsewhere. E.g. when your *remove* something. But when you add something that's already tracked by the `@since` of the method/var/hook etc.

Reply:
got it. Wrote this in my notes 📓